optimade/models/baseinfo.py

```diff
@@ -23,10 +23,17 @@ class AvailableApiVersion(BaseModel):
         "The version number string MUST NOT be prefixed by, e.g., 'v'.",
     )
 
+    @validator("url")
```

Comment:
Ah, I'd also done this and added a validator to check that the versions are compatible.

Reply:
Ah, sorry - I may have become a bit impatient :sweat_smile: sorry